### PR TITLE
BashSupport demo: parsing errors when more than one input line

### PR DESCRIPTION
**Pull request by alcortesm**

### CODE REVIEW

File: native/build.gradle [hunk 1]
```diff
@@ -0,0 +1,19 @@
+group 'tech.sourced'
```
Comment (smola) [inline native/build.gradle]: `bblfsh`

File: native/build.gradle [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+version '1.0-SNAPSHOT'
+
+apply plugin: 'java'
+apply plugin: 'application'
+
+mainClassName = "tech.sourced.Main"
```
Comment (smola) [inline native/build.gradle]: `bblfsh.bash.Main`

File: native/src/main/java/tech/sourced/LexerDemo.java [hunk 3]
```diff
@@ -0,0 +1,38 @@
+package tech.sourced;
+
+import com.ansorgit.plugins.bash.lang.BashVersion;
+import com.ansorgit.plugins.bash.lang.lexer.BashLexer;
+import com.intellij.psi.tree.IElementType;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class LexerDemo {
```
Comment (smola) [inline native/src/main/java/tech/sourced/LexerDemo.java]: Is this class really needed?

File: native/src/main/java/tech/sourced/Main.java [hunk 4]
```diff
@@ -0,0 +1,24 @@
+package tech.sourced;
+
+import com.ansorgit.plugins.bash.lang.BashVersion;
+import org.apache.commons.lang.StringEscapeUtils;
+
+public class Main {
+    public static void main  (String args[]) {
```
Comment (smola) [inline native/src/main/java/tech/sourced/Main.java]: You can convert this to a test. Check the `java-driver` for examples.

File: native/build.gradle (hunk 1 above)
Comment (alcortesm) [inline native/build.gradle]: done.

File: native/build.gradle (hunk 2 above)
Comment (alcortesm) [inline native/build.gradle]: done.

File: native/src/main/java/tech/sourced/LexerDemo.java (hunk 3 above)
Comment (alcortesm) [inline native/src/main/java/tech/sourced/LexerDemo.java]: It is no longer needed for the bblfsh driver, it only made sense when this was a demo of BashSupport. Removed.

File: native/src/main/java/tech/sourced/Main.java (hunk 4 above)
Comment (alcortesm) [inline native/src/main/java/tech/sourced/Main.java]: done.

**Comment by alcortesm** [comment]
WTF, I didn't want to merge this!

**Pull request merged**